docs/snippets/common/storybook-addon-toolkit-tsup-config.ts.mdx

```diff
@@ -1,22 +1,98 @@
 ```ts
 // tsup.config.ts
 
-import { defineConfig } from 'tsup';
-
-export default defineConfig((options) => ({
-  entry: ['src/index.ts', 'src/preview.ts', 'src/manager.ts'],
-  splitting: false,
-  minify: !options.watch,
-  format: ['cjs', 'esm'],
-  dts: {
-    resolve: true,
-  },
-  treeshake: true,
-  sourcemap: true,
-  clean: true,
-  platform: 'browser',
-  esbuildOptions(options) {
-    options.conditions = ['module'];
-  },
-}));
+import { defineConfig, type Options } from "tsup";
```

Comment:
I'm unsure if this code snippet is too long now, and we should just link to the tsup config in the addon-kit and not have any code snippet at all. Then we wouldn't also need to keep the two in sync.

Reply:
Yeah, that's pretty gnarly. And it's just demonstrating the default config, right?

I vote to get rid of it in the docs. 👍